packages/drizzle/src/queries/parseParams.ts

```diff
@@ -285,6 +285,39 @@ export function parseParams({
                   break
                 }
 
+                if (field.type === 'point') {
+                  switch (operator) {
+                    case 'near': {
+                      const [lng, lat, maxDistance, minDistance] = queryValue as number[]
+
+                      let constraint = sql`ST_DWithin(ST_Transform(${table[columnName]}, 3857), ST_Transform(ST_SetSRID(ST_MakePoint(${lng}, ${lat}), 4326), 3857), ${maxDistance})`
```

Comment:
Is it possible to reach this `switch` block from sqlite and blow up? Since it is in the `drizzle` package I'd expect to have  `if(this.name=== 'postgres') { //....` to be sure.

I know this won't include sqlite just yet but when we do we'll need separate handling when some day we do.

Reply:
✅